### PR TITLE
docs(spin): `spin` + `roer`/MPT clarification.

### PR DESCRIPTION
**Pull request by jtk54**
Small clarification doc about `spin` and `roer` + MPT. Let me know what you think.

### CODE REVIEW

File: guides/spin/comparison/index.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+## Foreward
+
+This document is meant to describe the current state of managing
+pipeline and application config in Spinnaker 'as is', and describe
+a bit about future threads of work in this area.
+
+## `roer` and Managed Pipeline Templates
+
+### What it is
```
Comment (demobox) [inline guides/spin/comparison/index.md]: A two-column table seems like the ideal format for this. I know Markdown isn't great with tables, but perhaps in this case it's worth "shelling out" to some HTML?

/cc @dorbin

File: guides/spin/comparison/index.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
+---
+layout: single
+title:  "Spin and Roer + Managed Pipeline Templates"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+## Foreward
```
Comment (demobox) [inline guides/spin/comparison/index.md:10]: Consider an intro along the lines of:

"Users are often interested in learning about how to implement an 'as-code' approach for Spinnaker, whether in conjunction with automating onboarding for teams, or just managing and versioning Spinnaker configuration.

A related question that also comes up is what the recommended way is to interact with Spinnaker programmatically, especially with authentication is enabled.

This document intends to describe two Spinnaker features that usually feature in the responses to those two questions: Managed Pipeline Templates (MPT)/`roer`, and the new `spin` CLI.

..."

File: guides/spin/comparison/index.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+MPT and `roer` are scoped strictly to make managing Spinnaker pipelines
+easier. It is not a generic solution to make API calls to Spinnaker.
+
+### Where it's going
+
+MPT and `roer` are not going away anytime soon. Plenty of folks
+happily use `roer` and MPT.
+
+That being said, it's not actively being developed -- we are
```
Comment (demobox) [inline guides/spin/comparison/index.md:47]: Worth going a little further to say that some aspects of MPT, especially the in-server template rendering, have posed more technical challenges than expected? That would hopefully help justify why future approaches may shy away from continuing to support server-side templating.

File: guides/spin/comparison/index.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+## Future Work
+
+We have gained valuable experience and insight from writing `roer`, MPT, and `spin`.
+
+We'll continue to grow `spin`'s adoption by expanding API coverage, hardening, and
+improving usability.
+
+We'll continue support for MPT and `roer`, while using the lessons we learned to
+inform design work on the next generation solution for managing Spinnaker pipelines at scale.
```
Comment (demobox) [inline guides/spin/comparison/index.md]: Perhaps worth adding that pipeline management is topic that we are actively looking into, that ideas/contributions in this area are encouraged, and that the "solution" to the issue will involve features other than just `spin`.

File: guides/spin/comparison/index.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+not adding new features.
+
+## `spin`
+
+### What it is
+
+`spin` is a CLI to interact with Spinnaker via the Gate API. It is
+currently in alpha, so the breadth of the interface is limited to managing
+pipeline and application config. Since `spin` is meant to be an API wrapper,
+it accepts JSON arguments and submits authenticated calls to Gate.
```
Comment (lwander) [inline guides/spin/comparison/index.md]: JSON "payloads" rather than "arguments" I think

File: guides/spin/comparison/index.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+for Orca to orchestrate.
+
+## Future Work
+
+We have gained valuable experience and insight from writing `roer`, MPT, and `spin`.
+
+We'll continue to grow `spin`'s adoption by expanding API coverage, hardening, and
+improving usability.
+
+We'll continue support for MPT and `roer`, while using the lessons we learned to
```
Comment (lwander) [inline guides/spin/comparison/index.md]: Unless the plan changed, I would be clear here that this is bug-fix support -- the next generation of MPT & roer will likely be different tools

File: guides/spin/comparison/index.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
+---
+layout: single
+title:  "Spin and Roer + Managed Pipeline Templates"
+sidebar:
+  nav: guides
```
Comment (demobox) [inline guides/spin/comparison/index.md:5]: @dorbin @jtk54 Is this something for the website or (given that it's likely a temporary-ish position statement) perhaps rather for community.spinnaker.io

**Review by robzienert — COMMENTED** [review]
Great clarification. Thanks for putting this together.

File: guides/spin/comparison/index.md (hunk 7 above)
Comment (dorbin) [reply guides/spin/comparison/index.md:5]: Agree that community.spinnaker.io (or blog.spinnaker.io?) would be better.

Comment (jtk54) [reply guides/spin/comparison/index.md:5]: I addressed all anph and lwander comments, exporting this to a Google doc for ddorbin to comment. Will generate the blag post from there and circle back to include it here once it's ship-shape. Closing this PR, will open another later that references the blog.

**Pull request closed without merging**